src/main/scala/play/dev/filewatch/FileWatchService.scala

```
@@ -1,142 +0,0 @@
-          catch {
-            case e: Throwable =>
-              logger.warn("Error loading Mac OS X watch service: " + e.getMessage)
-              logger.trace(e)
-              polling(pollDelayMillis)
-          }
-        // Fall back to polling watch service
-        case _ => polling(pollDelayMillis)
-      }
-
-      def watch(filesToWatch: Seq[File], onChange: () => Unit) = delegate.watch(filesToWatch, onChange)
-
-      override def toString = delegate.toString
-    }
-
-  def defaultWatchService(targetDirectory: File, pollDelayMillis: Int, logger: LoggerProxy): FileWatchService =
-    defaultWatchService(targetDirectory, pollDelayMillis, logger, disableFileHashCheck = false)
-
-  def jdk7(logger: LoggerProxy, disableFileHashCheck: Boolean): FileWatchService =
-    default(logger, isMac = false, disableFileHashCheck)
-
-  def jdk7(logger: LoggerProxy): FileWatchService = jdk7(logger, disableFileHashCheck = false)
-
-  def mac(logger: LoggerProxy, disableFileHashCheck: Boolean): FileWatchService =
-    default(logger, isMac = true, disableFileHashCheck)
-
-  def mac(logger: LoggerProxy): FileWatchService = mac(logger, disableFileHashCheck = false)
-
-  def polling(pollDelayMillis: Int): FileWatchService = new PollingFileWatchService(pollDelayMillis)
-
-  def optional(watchService: Try[FileWatchService]): FileWatchService =
-    new OptionalFileWatchServiceDelegate(watchService)
-
-  def default(logger: LoggerProxy, isMac: Boolean, disableFileHashCheck: Boolean): FileWatchService =
-    new DefaultFileWatchService(logger, isMac, disableFileHashCheck)
-
-  def default(logger: LoggerProxy, isMac: Boolean): FileWatchService =
-    default(logger, isMac, disableFileHashCheck = false)
-}
-
-/**
- * Watch service that delegates to a try. This allows it to exist without reporting an exception unless it's used.
- */
-class OptionalFileWatchServiceDelegate(val watchService: Try[FileWatchService]) extends FileWatchService {
```

I completely removed this class and `optional` method above. Using it with `Optional` does not make any sense at all.
This whole optional thing only existed because it was by the jnotify watch service, which I removed before in #119 already, and that was because the `JNotifyFileWatchService` could throw an exception [_on initialization_](https://github.com/playframework/play-file-watch/commit/a8a22e90945c9403fb91ade9b35054dd6e697d3c#diff-63ac30707156ea0a1a0fa888763aad966eecac49296a14e6d0f81662d5c9b89aR70-R75), to that's why this optional was introducted to wrap it in Try and not fail early when the service gets loaded.